src/appliance-onboarding-script/appliance_setup/run.py

```diff
@@ -18,6 +18,43 @@
 from avs.avsarconboarder.deleter.arcadon.arc_addon_deleter import ArcAddonDeleter
 from pkgs._utils import confirm_prompt
 
+def logger_setup(logLevel = logging.INFO):
```

Comment:
Should we create a separate folder for these log files? Also does it make sense to keep log files for all the onboarding attempts? Something like log_<Current_Timestamp>?

Reply:
updated